### PR TITLE
Satellite orbit and rotation

### PR DESCRIPTION
**Pull request by JiliTheSpaceboy**
This pull request primarily focuses on updating orbits of satellites of objects in dwarfplanets.ssc and outersys.ssc, though it also contains new sources for existing info, polishes, and so on.

### CODE REVIEW

File: data/dwarfplanets.ssc [hunk 1]
```diff
@@ -704,25 +704,34 @@ ReferencePoint "Orcus-Vanth" "Sol"
...
+		MeridianAngle	135.03
 	}
 	LunarLambert	0.5
 	GeomAlbedo	0.231
 	InfoURL	"https://en.wikipedia.org/wiki/90482_Orcus"
 }
 
+# Orbit from Grundy et al. (2019), Icarus 334, 62-78
+# "Mutual orbit orientations of transneptunian binaries"
+# https://ui.adsabs.harvard.edu/abs/2019Icar..334...62G/abstract
```
Comment (pedro-fixingstuff) [inline data/dwarfplanets.ssc:734]: May I suggest that this freely-accessible link be used instead? http://www2.lowell.edu/users/grundy/abstracts/2019.TNB_orbits.html
(same applies to the other instances where this paper is cited)

File: data/outersys.ssc [hunk 2]
```diff
@@ -530,7 +546,7 @@ ReferencePoint "Lempo-Hiisi" "Sol"
 # Colors from Grundy et al. (2015), Icarus 257, p. 130-138
 # "The mutual orbit, mass, and density of the large transneptunian binary system Varda and Ilmarë"
 # https://ui.adsabs.harvard.edu/abs/2015Icar..257..130G/abstract
-ReferencePoint "Varda-Ilmare" "Sol"
+ReferencePoint "Varda-Ilmar\u00EB" "Sol"
```
Comment (pedro-fixingstuff) [inline data/outersys.ssc:549]: Is there a need to do this on current versions of Celestia? AFAIK, with the intrduction of TTF fonts, it should be possible to input special characters directly in data files.

Comment (JiliTheSpaceboy) [reply data/outersys.ssc:549]: No idea. I can definitely just go with inputting the special character, but I don't know if that breaks stuff somewhere.

File: data/outersys.ssc [hunk 3]
```diff
@@ -633,15 +651,21 @@ ReferencePoint "Varda-Ilmare" "Sol"
 		ArgOfPericenter	346.87585252865
 		MeanAnomaly	344.208559857227
 	}
+	BodyFrame	{ EquatorJ2000 {} }
 	UniformRotation
 	{
 		Period	11.05
+		Inclination	133.9981188
+		AscendingNode	247.8337901
```
Comment (pedro-fixingstuff) [inline data/outersys.ssc:659]: Too much precision for what the source data allows?

File: data/dwarfplanets.ssc (hunk 1 above)
Comment (JiliTheSpaceboy) [reply data/dwarfplanets.ssc:734]: Sounds good; suggestion added and will be in the next commit.

File: data/outersys.ssc (hunk 3 above)
Comment (JiliTheSpaceboy) [reply data/outersys.ssc:659]: Fixed; they are now 134 and 248 respectively.

**Review by SevenSpheres — APPROVED** [review]

**Pull request merged**